app/controllers/api/v1/orders_controller.rb

```diff
@@ -38,33 +39,30 @@ def convert_json_to_params
   end
 
   def order_params
-    @json_params.require(:order).permit(:user_id, :status, :order_time, :pickup_time, :menu_items)
+    @json_params.require(:order).permit(:status, :order_time, :pickup_time)
```

Comment:
should we use `params.require...` here instead of `@json_params.require...` ? 
Because we have `before_action :convert_json_to_params, only: [:create, :update]`